include/rpi/hardware-control-implementation.h

```diff
@@ -15,20 +15,21 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-#ifndef RPI_GPIO_H
-#define RPI_GPIO_H
+#ifndef RPI_CONTROL_H
```

Comment:
simiilar here: let's derive the include-guard from the filename `RPI_HARDWARE_CONTROL_IMPLEMENTATION_H`